tests/playwright/src/runner/podman-desktop-runner.ts

```diff
@@ -55,15 +55,25 @@ export class PodmanDesktopRunner {
     try {
       // start the app with given properties
       this._running = true;
+      console.log(`### STARTING PODMAN DESKTOP APP ###`);
+      console.log(`Electron app launch options: `);
+      Object.keys(this._options).forEach(key => {
+        console.log(`${key}: ${(this._options as { [k: string]: string })[key]}`);
+      });
       this._app = await electron.launch({
         ...this._options,
       });
       // setup state
       this._page = await this.getElectronApp().firstWindow();
+      const exe = this.getElectronApp().evaluate(async ({ app }) => {
+        return app.getPath('exe');
+      });
+      console.log(`The Executable Electron app. file: ${exe}`);
```

Comment:
it looks like it's not only a move there

Reply:
Done